src/index.js

```diff
@@ -73,13 +73,13 @@ function validateRepeatRule(rrule) {
 }
 
 /**
- * Helper function for appending CRLF at start and end of file according to RFC rules.
+ * Helper function for appending CRLF at end of file according to RFC rules.
  * @function
  * @param {string} string - iCalendar source string
  * @return {string}
  */
 function addCRLF(string) {
-  return `\n${string}\n`
+  return `${string}\n`
```

Comment:
Name of this function doesn't match what it does

Reply:
i changed the name of the function